test/pages/register-a-beacon/additional-beacon-use.test.tsx

```diff
@@ -57,4 +61,45 @@ describe("AdditionalBeaconUse page", () => {
     expect(within(content).getByText(new RegExp(use2.activity, "i")));
     expect(within(content).getByText(new RegExp(use2.purpose, "i")));
   });
+
+  it("given a currentUseIndex, sends the user back down the editing route for that use", () => {
+    const currentUseIndex = 1;
+    render(
+      <AdditionalBeaconUse
+        uses={[getMockUse(), getMockUse()]}
+        currentUseIndex={currentUseIndex}
+      />
+    );
+
+    expect(screen.getByRole("link", { name: "Back" })).toHaveAttribute(
+      "href",
+      formatUrlQueryParams(PageURLs.moreDetails, { useIndex: currentUseIndex })
+    );
+  });
+
+  describe("getServerSideProps()", () => {
+    it("given a non-existent currentUseIndex, throws an error", () => {
+      const mockRegistration = {
+        getRegistration: jest
+          .fn()
+          .mockReturnValue({ model: "ASOS", uses: [getMockUse()] }),
+      };
+      const nonExistentUseIndex = "1";
+      const context = {
+        query: {
+          useIndex: nonExistentUseIndex,
+        },
+        container: {
+          getCachedRegistration: jest.fn().mockResolvedValue(mockRegistration),
+        },
+        req: {
+          cookies: {
+            [formSubmissionCookieId]: "test-submission-id",
+          },
+        },
+      };
+
+      expect(() => getServerSideProps(context as any)).rejects.toThrow();
+    });
```

Comment:
@matthew-a-carr and I discussed default behaviour for this page.  I decided against specifying default behaviour as I felt like I was loading a foot gun as I was writing it, making it so that if no currentUseIndex is provided it sends the user down either the first or last use case.  Thinking through the various options and implications at edge cases, nothing was leaping out as an obvious default choice.  

So I decided to not set a default behaviour for no currentUseIndex; now pages that redirect here must make a decision about the current use (and where the back button should go).  Intending to keep an eye on this as it develops.